src/sentry/integrations/jira/integration.py

```diff
@@ -394,13 +396,19 @@ def get_create_issue_config(self, group, **kwargs):
 
         # TODO(jess): are we going to allow ignored fields?
         # ignored_fields = (self.get_option('ignored_fields', group.project) or '').split(',')
-        ignored_fields = set()
+
+        # hide sprint, epic link, parent and linked issues fields because they don't work
+        # since sprint and epic link are "custom" we need to search fo rthem by name
+        ignored_fields = set(
+            k for k, v in six.iteritems(issue_type_meta['fields']) if v['name'] in {'Sprint', 'Epic Link'}
```

Comment:
ooo, rare set syntax.

Can you suck this out into a constant at the top

`HIDDEN_ISSUE_FIELDS= {'Sprint', 'Epic Link'}`

Reply:
yeah, i thought about doing this, but it's also sort of weird that then i'd also have to have another global for the ones we can should use keys for :-/

Reply:
how about 

```py
HIDDEN_ISSUE_FILEDS = {
  'by_key': ['parent', 'issuelinks']
  'by_custom_name': ['Sprint', 'Epic Link'],
}
```